cli/snyk.js

```diff
@@ -70,10 +70,12 @@ module.exports = class Snyk {
 };
 
 function getSeverities(minimumSeverity) {
-    if (minimumSeverity && minimumSeverity.toLowerCase() === 'high') {
-        return ['high'];
+    if (minimumSeverity && minimumSeverity.toLowerCase() === 'critical') {
+        return ['critical', 'high'];
```

Comment:
I think you meant to only include `critical` in this array right?

```suggestion
        return ['critical'];
```

Reply:
Yep, good catch thanks!